node/node.go

```diff
@@ -389,59 +389,71 @@ var (
 // verify message signature
 func (node *Node) validateShardBoundMessage(
 	ctx context.Context, payload []byte,
-) (*msg_pb.Message, *bls.PublicKey, error) {
+) (*msg_pb.Message, *bls.PublicKey, bool, error) {
 	var (
 		m         msg_pb.Message
 		senderKey *bls.PublicKey
 	)
 	entryTime := time.Now()
-	defer utils.SampledLogger().Info().Str("cost", time.Now().Sub(entryTime).String()).Msg("[cost:validate_shard_bound_message]")
+	defer utils.SampledLogger().Debug().Str("cost", time.Now().Sub(entryTime).String()).Msg("[cost:validate_shard_bound_message]")
 
 	if err := protobuf.Unmarshal(payload, &m); err != nil {
 		atomic.AddUint32(&node.NumInvalidMessages, 1)
-		return nil, nil, errors.WithStack(err)
+		return nil, nil, true, errors.WithStack(err)
 	}
+	if node.Consensus.IsLeader() {
+		switch m.Type {
+		case msg_pb.MessageType_ANNOUNCE, msg_pb.MessageType_PREPARED, msg_pb.MessageType_COMMITTED:
+			return nil, nil, true, nil
```

Comment:
We don't need the boolean.  can just return a new error here "Wrong message type" or "Unacceptable message type"

Reply:
It's not a real error though. Those are valid messages. that's why I want to be explicit to ignore it.